layers/ChapterLayer.js

```diff
@@ -0,0 +1,46 @@
+/* global ChapterLayer: true, cc, ccui, res
+*/
+ChapterLayer = ccui.Widget.extend({
```

Comment:
Are we not adding namespaces to these global variables ?

Reply:
We haven't done that yet. Should I start with this PR?

Reply:
Will update it with this PR

Reply:
done